src/main/java/com/example/spring/dto/book/BookDtoWithoutCategoryIds.java

```diff
@@ -0,0 +1,14 @@
+package com.example.spring.dto.book;
+
+import java.math.BigDecimal;
+
+public record BookDtoWithoutCategoryIds(
+         Long id,
+         String title,
+         String author,
+         String isbn,
+         BigDecimal price,
+         String description,
+         String coverImage

```

Comment:
the same here

Reply:
the same here
no need to validate because we get info from the db (it is already validated)